strawberryfields/configuration.py

```diff
@@ -14,139 +14,231 @@
 r"""
 This module contains the :class:`Configuration` class, which is used to
 load, store, save, and modify configuration options for Strawberry Fields.
+
+Configuration options
+*********************
+
+.. note::
+    The following configuration options are taken into consideration:
+
+        * **authentication_token (str)** (*required*): the token used for user authentication
+        * **hostname (str)** (*optional*): the name of the host to connect to
+        * **use_ssl (bool)** (*optional*): specifies if requests should be sent using SSL
+        * **port (int)** (*optional*): the port to be used when connecting to the remote service
+        * **debug (bool)** (*optional*): determines if the debugging mode is requested
```

Comment:
The bullet point list needs to be de-indented to match the indentation level of `The following...` above.